inc/dbmysql.class.php

```diff
@@ -381,7 +381,12 @@ function field_flags($result, $field) {
     * @return list of tables
    **/
    function list_tables($table="glpi_%") {
-      return $this->query("SHOW TABLES LIKE '".$table."'");
+      return $this->query(
+         "SELECT TABLE_NAME FROM information_schema.`TABLES`
+             WHERE TABLE_SCHEMA = '{$this->dbdefault}'
+                AND TABLE_TYPE != 'VIEW'
```

Comment:
There is no reason... Those two values are the only ones than can be present; so it does not change anything right now :)